pkg/render/manifests.go

```diff
@@ -12,14 +12,14 @@ import (
 )
 
 // RenderClusterManifests renders manifests for a hosted control plane cluster
-func RenderClusterManifests(params *api.ClusterParams, pullSecretFile, outputDir string, externalOauth, includeRegistry bool) error {
+func RenderClusterManifests(params *api.ClusterParams, pullSecretFile, outputDir string, externalOauth, includeRegistry, konnectivityEnabled bool) error {
 	releaseInfo, err := release.GetReleaseInfo(params.ReleaseImage, params.OriginReleasePrefix, pullSecretFile)
 	if err != nil {
 		return err
 	}
 	includeMetrics := len(params.ROKSMetricsImage) > 0
 	ctx := newClusterManifestContext(releaseInfo.Images, releaseInfo.Versions, params, outputDir)
-	ctx.setupManifests(externalOauth, includeRegistry, includeMetrics)
+	ctx.setupManifests(externalOauth, includeRegistry, includeMetrics, konnectivityEnabled)
```

Comment:
I recommend calling this `includeKonnectivity` to align with the other parameters.